lib/seahorn/BvOpSem2.cc

```diff
@@ -653,6 +654,25 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     m_ctx.setMemReadRegister(Expr());
   }
 
+  void visitResetModified(CallSite CS) {
+    if (!m_ctx.getMemReadRegister() || !m_ctx.getMemWriteRegister()) {
+      LOG("opsem",
+          ERR << "No read/write register found - check if corresponding"
+                 "shadow instruction is present.");
+      m_ctx.setMemReadRegister(Expr());
```

Comment:
same comment as before. If this is needed, it is masking a deeper problem.

Reply:
or maybe this is just resetting the variables to null. In this case, add a comment. I though they are set to nondet.

Reply:
See comment above.